editor/src/components/canvas/canvas-strategies/strategies/shared-move-strategies-helpers.ts

```diff
@@ -354,7 +360,7 @@ export function moveInspectorStrategy(
         intendedBounds.push(...moveCommandsResult.intendedBounds)
       }
       commands.push(pushIntendedBoundsAndUpdateGroups(intendedBounds, 'live-metadata'))
-      commands.push(setElementsToRerenderCommand(selectedElementPaths))
```

Comment:
do these removed setElementsToRerenderCommands have a counterpart which is setting the elementsToRerender value?

Reply:
They already appear to from some surrounding logic which also passes the selected paths through as the elements to render.